docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Intel Platform Software for Windows (PSW) v2.7](http://registrationcenter-download.intel.com/akdlm/irc_nas/16464/Intel%20SGX%20PSW%20for%20Windows%20v2.7.100.2.exe)
 
+The latest PSW should be installed automatically on a Windows machine with Windows
+version no lower than 1709. To check your Windows version, run `winver` on the
+command line.
+
```

Comment:
Can we also said that the following instruction is for older version of windows and explicitly mention Windows Server 2016? According to [here](https://cloudblogs.microsoft.com/windowsserver/2017/10/26/faq-on-windows-server-version-1709-and-semi-annual-channel/), Windows Server 2016 seems unlikely to have version above 1709.

Also, do we need to also manual installation steps for Windows Server 2019 as mentioned in #2267? Or should we also add instructions for checking if the PSW is up-to-date (if not, the instruction/pointer for update)?

Reply:
Will add Windows Server 2019 manual installation instructions in a separate PR.